internal/views/publishers/new.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Criar Editora</title>
+</head>
+<body>
+    <h1>Criar Editora</h1>
+    <form action="/publishers" method="post">
+        <label for="name">Nome</label>
+        <input type="text" id="name" name="name" required>
+        <button type="submit">Criar</button>
+    </form>
+</body>
+</html>
```

Comment:
Esse símbolo no final indica que não tem uma linha em branco no final do arquivo. A gente usa o [EditorConfig](https://editorconfig.org/) para formatar os arquivos com o mesmo padrão. Instala o plug-in do EditorConfig para o seu editor e tenta salvar esse arquivo para ver se a linha é adicionada.